src/test/resources/simple/not-allowed-url-bag/fetch.txt

```diff
@@ -0,0 +1 @@
+ftp://dans.knaw.nl/nl/over/organisatie-beleid/informatiemateriaal/Software_Sustainability_DANS_NLeSC_2016.pdf  555292  data/brochure/Software_Sustainability_DANS_NLeSC_2016.pdf
```

Comment:
That is better indeed. Then you know for sure that the file doesn't suddenly disapperar.